tests/conftest.py

```diff
@@ -48,6 +36,10 @@ def pytest_configure(config):
 register(AreaSettingsFactory)
 
 
+def pytest_configure():
+    Celery(task_always_eager=True)
+
```

Comment:
I wouldn't create another celery here for a+, but I'd add the celery_config in the fixtures:
https://docs.celeryq.dev/en/stable/userguide/testing.html#session-scope

Reply:
Agreed to leave this option open for future